azkaban-web-server/src/web/js/azkaban/view/schedule-sla.js

```diff
@@ -189,6 +189,16 @@ azkaban.ChangeSlaView = Backbone.View.extend({
       var duration = rFlowRule.cells[2].firstChild.value;
       var email = rFlowRule.cells[3].firstChild.checked;
       var kill = rFlowRule.cells[4].firstChild.checked;
+      if (!email && !kill) {
+        alert("Unable to create SLA Action. Please set either Kill or Email"
+            + " action");
+        return false;
+      }
+      if (!email && slaEmails === "") {
```

Comment:
Please use:
```
var slaEmails = $('#slaEmails').val().trim();
```
We might want to _trim_ the value to discard white spaces. 
Also, we're not validating the emails specified but is fine to do it in another time as this hole SLA code should probably be revisited and refactored to use jQuery or Backbone instead of plain Javascript.

Reply:
For now, I will just trim the white spaces.